examples/tutorials/core/plot_3_handle_probe_info.py

```diff
@@ -47,16 +47,23 @@
 plot_probe(recording_2_shanks.get_probe())
 
 ###############################################################################
-# Now let's check what we have loaded. The `group_mode='by_shank'` automatically
+# Now let's check what we have loaded. The :code:`group_mode='by_shank'` automatically
 # sets the 'group' property depending on the shank id.
-# We can use this information to split the recording into two sub-recordings:
+# We can use this information to split the recording into two sub-recordings.
+# We can acccess this information either as a dict with :code:`outputs='dict'` (default)
```

Comment:
```suggestion
# We can access this information either as a dict with :code:`outputs='dict'` (default)
```

Reply:
Thanks that's what I get for typing fast!